### PR TITLE
dd-trace-php b3 headers public documentation

### PR DESCRIPTION
**Pull request by krakjoe**
### What does this PR do?
Add public PHP documentation for propagation (b3) headers

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/trace_collection/library_config/php.md [hunk 1]
```diff
@@ -288,6 +288,28 @@ The IP header to be used for client IP collection, for example: `x-forwarded-for
...
+`DD_TRACE_PROPAGATION_STYLE_EXTRACT`
+: **INI**: `datadog.trace.propagation_style_extract`<br>
+**Default**: `Datadog,B3,B3 single header`<br>
+Propagation styles to use when extracting tracing headers. Currently supported styles are:
+
+  - B3[7]
+  - B3 single header[8]
+  - Datadog
+
+Multiple styles may be used, comma separated.
```
Comment (maycmlee) [inline content/en/tracing/trace_collection/library_config/php.md:311]:
```suggestion
If using multiple styles, comma separate them.
```

**Review by maycmlee — CHANGES_REQUESTED** [review]
Hey @krakjoe! Left some suggestions and also fixed the links.

By the way, you should be able to clone the documentation repo, rather than fork, so that when you open a PR, a preview of your changes is automatically created.

**Comment by krakjoe** [comment]
> Hey @krakjoe! Left some suggestions and also fixed the links.
> 
> By the way, you should be able to clone the documentation repo, rather than fork, so that when you open a PR, a preview of your changes is automatically created.

Thank you for the hint, and suggestions :)

File: content/en/tracing/trace_collection/library_config/php.md [hunk 2]
```diff
@@ -288,6 +288,28 @@ The IP header to be used for client IP collection, for example: `x-forwarded-for
...
+`DD_TRACE_PROPAGATION_STYLE_INJECT`
+: **INI**: `datadog.trace.propagation_style_inject`<br>
+**Default**: `Datadog`<br>
+Propagation styles to use when injecting tracing headers. The supported styles are:
+
+  - [B3][7]
+  - [B3 single header][8]
+  - Datadog
+
+If using multiple styles, comma separate them.
```
Comment (maycmlee) [inline content/en/tracing/trace_collection/library_config/php.md:300]:
```suggestion
```

File: content/en/tracing/trace_collection/library_config/php.md [hunk 3]
```diff
@@ -288,6 +288,28 @@ The IP header to be used for client IP collection, for example: `x-forwarded-for
   ```
   Regular expression used to obfuscate the query string included as part of the URL. Added in version `0.76.0`.
 
+`DD_TRACE_PROPAGATION_STYLE_INJECT`
+: **INI**: `datadog.trace.propagation_style_inject`<br>
+**Default**: `Datadog`<br>
+Propagation styles to use when injecting tracing headers. The supported styles are:
```
Comment (maycmlee) [inline content/en/tracing/trace_collection/library_config/php.md:294]:
```suggestion
Propagation styles to use when injecting tracing headers. If using multiple styles, comma separate them. The supported styles are:
```

File: content/en/tracing/trace_collection/library_config/php.md [hunk 4]
```diff
@@ -288,6 +288,28 @@ The IP header to be used for client IP collection, for example: `x-forwarded-for
...
+
+  - [B3][7]
+  - [B3 single header][8]
+  - Datadog
+
+If using multiple styles, comma separate them.
+
```
Comment (maycmlee) [inline content/en/tracing/trace_collection/library_config/php.md:301]:
```suggestion
```

File: content/en/tracing/trace_collection/library_config/php.md [hunk 5]
```diff
@@ -288,6 +288,28 @@ The IP header to be used for client IP collection, for example: `x-forwarded-for
...
+`DD_TRACE_PROPAGATION_STYLE_EXTRACT`
+: **INI**: `datadog.trace.propagation_style_extract`<br>
+**Default**: `Datadog,B3,B3 single header`<br>
+Propagation styles to use when extracting tracing headers. The supported styles are:
+
+  - [B3][7]
+  - [B3 single header][8]
+  - Datadog
+
+If using multiple styles, comma separate them.
```
Comment (maycmlee) [inline content/en/tracing/trace_collection/library_config/php.md:311]:
```suggestion
```

File: content/en/tracing/trace_collection/library_config/php.md [hunk 6]
```diff
@@ -288,6 +288,28 @@ The IP header to be used for client IP collection, for example: `x-forwarded-for
...
+
```
Comment (maycmlee) [inline content/en/tracing/trace_collection/library_config/php.md:312]:
```suggestion
```

File: content/en/tracing/trace_collection/library_config/php.md [hunk 7]
```diff
@@ -288,6 +288,28 @@ The IP header to be used for client IP collection, for example: `x-forwarded-for
...
+`DD_TRACE_PROPAGATION_STYLE_EXTRACT`
+: **INI**: `datadog.trace.propagation_style_extract`<br>
+**Default**: `Datadog,B3,B3 single header`<br>
+Propagation styles to use when extracting tracing headers. The supported styles are:
```
Comment (maycmlee) [inline content/en/tracing/trace_collection/library_config/php.md:305]:
```suggestion
Propagation styles to use when extracting tracing headers. If using multiple styles, comma separate them. The supported styles are:
```

**Review by maycmlee — COMMENTED** [review]
Hey @krakjoe! Is it okay to move that last line up? I'm having problems formatting it at the end (can't get it to flushed left for some reason). https://a.cl.ly/mXu2Rzgv

**Review by maycmlee — APPROVED** [review]

**Pull request merged**